Counties/Florida/Bay County/Scraper/utils/ScraperUtils.py

```diff
@@ -103,16 +141,37 @@ def parse_plea_type(plea_text: str):
     :param plea_text: Plea text in case dockets
     :return: 'Not Guilty', 'Guilty', 'Nolo Contendere', or None.
     """
-    if 'NOT' in plea_text:
-        plea = 'Not Guilty'
-    elif 'GUILTY' in plea_text:
-        plea = 'Guilty'
-    elif 'NOLO' in plea_text:
-        # Nolo-Contendere (no contest) plea
-        plea = 'Nolo Contendere'
-    else:
-        plea = None
-    return plea
+    try:
+        if 'NOT' in plea_text:
+            plea = 'Not Guilty'
+        elif 'GUILTY' in plea_text:
+            plea = 'Guilty'
+        elif 'NOLO' in plea_text:
+            # Nolo-Contendere (no contest) plea
+            plea = 'Nolo Contendere'
+        else:
+            plea = None
+        return plea
+    except TypeError:
```

Comment:
It seems weird that this should occur. Is it because None could be passed in? If so, it may be cleaner to do a `plea_text = plea_text or ''` (to see to `''` if it's empty) before we perform our comparisons.
If that's not the case, can you add a comment explaining why we need to do this?

Reply:
That is the reason, thanks, that change to handle None cases is much cleaner :)